src/lib/file.js

```diff
@@ -76,7 +77,7 @@ export function isWatermarked(file) {
  * @return {boolean} Whether or not action is permitted
  */
 export function checkPermission(file, operation) {
-    return !!file && !!file.permissions && !!file.permissions[operation];
+    return getProp(file, `permissions.${operation}`, false);
```

Comment:
Could also use getProp(file, permissions, operation) without combined string

Reply:
Went to custom lodash.get implementation because lodash.get is 4.8KB un-gzipped.